regression/ansi-c/struct5/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 main.c
-
+--i386-win32
 ^EXIT=0$
```

Comment:
Seems unrelated?

Reply:
No, this is a relevant change. We previously wrongly supported the code in this test on all platforms: it uses anonymous (but tagged) union members.